plugins/aws/resoto_plugin_aws/resource/route53.py

```diff
@@ -217,5 +217,21 @@ class AwsRoute53ResourceRecordSet(AwsResource, BaseDNSRecordSet):
     record_traffic_policy_instance_id: Optional[str] = field(default=None)
     record_cidr_routing_config: Optional[AwsRoute53CidrRoutingConfig] = field(default=None)
 
+    def _keys(self) -> tuple:
+        if self._graph is None:
+            raise RuntimeError(f"_keys() called on {self.rtdname} before resource was added to graph")
+        return (
+            self.kind,
+            self.cloud().id,
+            self.account().id,
+            self.region().id,
+            self.zone().id,
+            self.dns_zone().id,
+            self.id,
+            self.name,
+            self.record_type,
+            self.record_set_identifier,
```

Comment:
Maybe the `id` should be defined as combination of `set_identifier` and `name`, so it becomes unique?

Reply:
The idea of the ID though is not to be unique but to be used for API requests like when the resource should be deleted. So we shouldn't just invent our own ids. Otherwise for any API requests we then have to split it back up. Which is extra bad if a user wants to use it as input for the `aws` cli command.

Reply:
I see.